virttest/cpu.py

```diff
@@ -1050,6 +1050,42 @@ def _make_up_pattern(flags):
     return cpu_support_model
 
 
+def parse_qemu_cpu_models_modern(help_text):
+    """
+    Get all cpu models from qemu -cpu help text.
+
+    Note: not completely supported for qemu version prior to v9.0.0
```

Comment:
Hi, @luckyh the behavior change commit is actually on top of qemu9.0-rc4, so I think QEMU versions that are rebased before rc4 won't be affected by it. This means that the legacy versions may include some qemu9.0 version. Not clear about the rebase rule, I think we'd better check it with yfu first.

Reply:
@aliang123 good point. However, whether the function is going to be invoked did not depend on any check on the qemu versions, instead, the only prerequisite is that the given `help_text` must contain the `"Available CPUs:"` pattern so that the model names could be identified and extracted afterwards. This is just a note for users who would like to call the new function directly and to expect it working properly in their use cases.